azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -94,6 +94,19 @@
   private final ArrayList<ObjectName> registeredMBeans = new ArrayList<ObjectName>();
   private MBeanServer mbeanServer;
 
+  public static boolean testing = false;
+
+  public AzkabanExecutorServer(ExecutorLoader executionLoader, FlowRunnerManager runnerManager, Props props,
```

Comment:
Ok I hope so. The reason for the pattern I've chosen is because of trying to avoid merge conflicts, if these changes would only be done in private fork. I'm all in for doing it properly now that it's going towards the public repo.